Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3
+
+WORKDIR /usr/src/app
+
+COPY . .
+
+CMD [ "python", "./main.py" ]
```

Comment:
No newline at end of file

Reply:
WTAF !!!!!! its just fucking 4 lines ......needed to get some fillers in...... don't you watch anime ?